packages/cozy-flags/README.md

```diff
@@ -47,8 +47,8 @@ const App = () => {
 ### Flags enabled on the server
 
 Feature flags can be served by the server and injected in the HTML page of an
-application. See https://docs.cozy.io/en/cozy-stack/settings/#get-settingsflags
-and https://docs.cozy.io/en/cozy-stack/client-app-dev/#good-practices-for-your-application.
+application. See <a href="https://docs.cozy.io/en/cozy-stack/settings/#get-settingsflags">https://docs.cozy.io/en/cozy-stack/settings/#get-settingsflags</a>
+and <a href="https://docs.cozy.io/en/cozy-stack/client-app-dev/#good-practices-for-your-application">https://docs.cozy.io/en/cozy-stack/client-app-dev/#good-practices-for-your-application</a>.
```

Comment:
I think this part of the doc should be put below the plugin as it is more of interest for advanced users.

Also, I think it'd be more readable if the links had a title and were written in markdown:

```
See [the stack documentation on flags])https://docs.cozy.io/en/cozy-stack/settings/#get-settingsflags) and [how to access flags at page load](https://docs.cozy.io/en/cozy-stack/client-app-dev/#good-practices-for-your-application).
```